conf/modules.d/arc.conf

```diff
@@ -26,25 +26,27 @@ arc {
   # If false, messages with empty envelope from are not signed
   allow_envfrom_empty = true;
   # If true, envelope/header domain mismatch is ignored
-  allow_hdrfrom_mismatch = false;
+  allow_hdrfrom_mismatch = true;
   # If true, multiple from headers are allowed (but only first is used)
-  allow_hdrfrom_multiple = false;
+  allow_hdrfrom_multiple = true;
```

Comment:
Why?

Reply:
You correct, that doesn't make sense. I misinterpreted the meaning of that
option.

Fixed.